packages/forms-web-app/src/dynamic-forms/journey.js

```diff
@@ -250,6 +250,15 @@ class Journey {
 		);
 	};
 
+	/**
+	 * Gets the url for the current question
+	 * @param {string} questionSegment - question segment name
+	 * @returns {string} url for the current question
+	 */
+	getCurrentQuestionUrlWithoutSection = (questionSegment) => {
+		return `${this.baseUrl}/${encodeURIComponent(questionSegment)}`;
```

Comment:
bit concerned we're not centralizing the logic in buildQuestionUrl, anything that relies on that function will fail right?

Reply:
Ah sorry, must've had a brain fart. Did this as the 'Change' / 'Answer' button on the task list/ check your answers page was adding the section back in to the url. Was thinking the getCurrentQuestionUrl wouldnt make it to the buildQuestionUrl due to the check of no section... 

Will change it